api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -33,6 +33,7 @@ public class ApiConstants {
     public static final String ANNOTATION = "annotation";
     public static final String API_KEY = "apikey";
     public static final String ARCHIVED = "archived";
+    public static final String ARCHITECTURE = "architecture";
```

Comment:
shortened `arch` is acceptable term, `/usr/bin/arch` returns the linux distro arch.